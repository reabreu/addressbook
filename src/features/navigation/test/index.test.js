import React from "react";
import { render } from "@testing-library/react";
import { Provider } from "react-redux";
import Search from "../";
import getStore from "../../../store";
import { BrowserRouter as Router } from "react-router-dom";

describe("<Navigation /> spec", () => {
  it("Matches snapshot", () => {
    const { asFragment } = render(
      <Router>
        <Provider store={getStore()}>
          <Search />
        </Provider>
      </Router>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
