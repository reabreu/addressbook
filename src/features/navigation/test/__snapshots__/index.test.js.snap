// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> spec Matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxirZ kXLBxR"
  >
    <nav
      class="sc-AxjAm eRCOtq"
    >
      <div
        class="sc-AxiKw eVuaxy"
      >
        <li
          class="sc-AxhCb cbjXGh"
        >
          <a
            class="sc-AxhUy liakIm"
            href="/"
          >
            Home
          </a>
        </li>
        <li
          class="sc-AxhCb cbjXGh"
        >
          <a
            class="sc-AxhUy liakIm"
            href="/settings"
          >
            Settings
          </a>
        </li>
        <li
          class="sc-AxhCb cbjXGh"
        >
          <input
            class="sc-AxgMl bMeUUL"
            placeholder="Search something here"
            value=""
          />
        </li>
      </div>
      <p>
        Active language is GB
      </p>
    </nav>
  </div>
</DocumentFragment>
`;
